contracts/contracts/governance/Governor.sol

```diff
@@ -189,6 +207,28 @@ contract Governor {
         emit ProposalExecuted(proposalId);
     }
 
+    function cancel(uint256 proposalId) public onlyGuardian {
+        ProposalState proposalState = state(proposalId);
+
+        require(
+            proposalState == ProposalState.Queued ||
+                proposalState == ProposalState.Pending,
+            "Governor::execute: proposal can only be cancelled if it is queued or pending"
+        );
+        Proposal storage proposal = proposals[proposalId];
+        proposal.eta = 1; // To mark the proposal as `Expired`
```

Comment:
Do you think we need a separate cancel state? I think expired is probably sufficient

Reply:
I don't see a need for it now. But can add it, if it's necessary

Reply:
Nah I don't think it is, was thinking out loud 👍